NEWS

```diff
@@ -1,3 +1,52 @@
+KLEE 1.4.0, 20 July 2017
+========================
+(Incorporating changes from 4 November 2016 up to and including 20 July 2017)
+
+- New CMake build system (@delcypher, @jirislaby)
+- Added support for vectorized instructions (@delcypher)
+- Fixed and documents BFS searcher behaviour (@MartinNowack, @ccadar)
+- Renames .pc files to .kquery files (@holycrap872)
+- Removed option --randomize-fork (@ccadar)
+- Changed preferred permissions from 0622 to the more standard 0644 in the
+POSIX model (@ccadar)
+- New names (systemtests and unittests) for the test targets in the build
+systems
+- Better support for MacOS (@andreamattavelli, @delcypher)
+- Enabled support for ASan builds of KLEE (@delcypher)
+- Support long long values in --stop-after-n-instructions for LLVM > 2.9
+(@andreamattavelli)
+- Teach KLEE to respect the requested memory alignment of globals and stack
+variables when possible (@delcypher)
+- Added new option --warnings-only-to-file which causes warnings to be written
+to warnings.txt only (@ccadar)
+- metaSMT improvements (@hoangmle)
+- KLEE-web improvements (@andronat, @helicopter88)
+- Fixed bug in the implementation of NotExpr (@delcypher)
+- Fixed a bug leading to data loss when writing into files (@ccadar, @gladtbx)
+- Some improvements and refactoring to the Expr library (@delcypher)
+- Added missing constant folding opportunity when handling constant arrays
+(@andreamattavelli, @delcypher)
+- Teach klee::getDirectCallTarget() to resolve weak aliases (@delcypher)
+- Fixed handling of internal forks (@gladtbx)
+- Improved replay using libkleeRuntest (@delcypher)
+- Added -debug-assignment-validating-solver feature to check the correctness
+of generated assignments (@delcypher)
+- Added -debug-z3-dump-queries, -debug-z3-validate-models and
+-debug-z3-verbosity options useful for debugging the interaction with Z3
+(@delcypher)
+- Added geq/lt-llvm- configs in lit (@jirislaby)
+- Work on supporting newer LLVM versions (@jirislaby)
```

Comment:
@ccadar 
Might be worth noting that

KLEE can now compile and pass its test suite when linking against LLVM 3.5 and 3.6. However this support is experimental and constructs in the newer versions of LLVM IR might not be supported. If you run into issues with particular instructions or instrinsics please report them. You can see a summary of know issues in https://github.com/klee/klee/issues/678